src/client/components/inputs/index.tsx

```diff
@@ -325,7 +325,8 @@ export const InputGroupDeleteButton = styled.button`
   }
 
   :hover {
-    background: ${colorsV2.coral100};
+    /* HEX value with opcaity (1a = 10% transparency) */
+    background-color: ${colorsV3.red500}1a;
```

Comment:
Since we don't have a translucent red in brand and this seems to be the only occurrences  
https://user-images.githubusercontent.com/6661511/144036700-b91377bd-7a68-47c2-8387-c770258eaa9b.mov

Reply:
No IE11 compatibility: https://caniuse.com/css-rrggbbaa 😅 

Did we decide on that? Let's write what the browser requirements are in Notion!

Reply:
Sorry not sorry IE11 🤪 We've said that we shouldn't support IE11, looking at the stats we have 0,09% of visitors on IE11 since 1/6. But let me know if you disagree :) This would be more of a graceful degradation since it's only for a hover state and not a breaking change.